.github/workflows/workflow.yml

```diff
@@ -143,16 +136,22 @@ jobs:
           source-url: "https://nuget.pkg.github.com/${{ env.OWNER }}/index.json"
       - name: Load Data into elasticsearch & prepare for tests
         env:
-          ELASTIC_SEARCH_HOST: http://localhost:${{ job.services.elasticsearch.ports[9200] }}
+          ELASTIC_SEARCH_HOST: http://localhost:9200
         run: |
-              ## Create test output and API logging location
-              mkdir -p integration-tests/target
+            # Sitewide search uses a synonyms list which is placed directly into the Elasticsearch config
+            # directory. It's not possible to do this with a service container as they are created before the job
+            # has performed its checkout step. We therefore have to manage Docker directly.
+            # Run only elasticsearch.
+            docker compose --file integration-tests/docker-sws-api/docker-compose.yml up --detach elasticsearch
```

Comment:
This should be its own workflow step to start docker. Then there should be a separate step to stop it cleanly. It should stop even if tests fail.

Reply:
Seems reasonable.